### ISSUE TITLE
Introduce keybindings for dev mode

### ISSUE THREAD

**Issue by goloroden** [thenativeweb/wolkenkit#637]
### What is this feature about?

When running the CLI in `dev` mode, it could be useful to have some keyboard shortcuts to run a number of actions, e.g.:

- `r` to restart the server
- `c` to open the console in the browser

However, these are just ideas, and before implementing this, we need to come up with a specific list of things.

### What needs to be done to implement this feature?

- [ ] Think about which commands could be useful
- [ ] Handle key strokes in the CLI
- [ ] Run the defined actions
- [ ] Show the available key bindings when starting the CLI

### What else should we know?

This issue was brought up by @schmuecker, so we should notify him once this has been done.

**Comment by goloroden** [comment]
@schmuecker First of all I would like to thank you again for bringing up this idea. This morning @yeldiRium and I talked about it and I would like to write you our feedback on it now. To make a long story short, we decided not to include this feature in wolkenkit. There are several reasons for this, which I would like to explain to you briefly in the following:

As for the restart, we believe that there are already enough tools to do it. An example of this would be [nodemon](https://www.npmjs.com/package/nodemon), which automatically restarts a Node.js process after every change to a file.

You may not want to restart after every single file change, but only from time to time. Here we see no advantage over the classic approach: Either way, you have to switch to the terminal and press a key. Whether you press `R` or `<Ctrl>+<C>` and then `Cursor Up` and `Enter` doesn't make too much difference in efficiency.

Last but not least, you have to take into account that you could leave the application in a non-executable state by making a file change - this would mean that the CLI would have to handle this as well, not executing the application but still waiting for a keystroke.

The bottom line is that we believe that the effort to implement this far outweighs the benefit, especially since - as I said - there are already solutions for an automatic restart.

The option to open the console with a keystroke is similar: we could print the address to the command line, from where it can be opened with a double-click. In our opinion, adding a key handler for this is not in proportion to the effort involved.

It should be noted that we would probably need an external library for the keyhandler, which in turn would increase the installation complexity of wolkenkit.

We hope you understand our reasons, and that you are not disappointed. We would be happy if we still get feedback from you regarding wolkenkit! 😊

**Comment by schmuecker** [comment]
Thanks for discussing the topic and reaching out!
It's reasonable to have decided against the keystrokes 👍
